src/main/java/brokeculator/parser/DeleteParser.java

```diff
@@ -0,0 +1,25 @@
+package brokeculator.parser;
+
+import brokeculator.command.DeleteCommand;
+import brokeculator.expense.ExpenseManager;
+
+public class DeleteParser {
+    private ExpenseManager expenseManager;
+
+    public DeleteParser(ExpenseManager expenseManager) {
+        this.expenseManager = expenseManager;
+    }
+
+    public DeleteCommand parseInput(String userInput) {
+        String[] userInputAsArray = userInput.split(" ");
```

Comment:
I feel like we should excessively handle possible user inputs. For example if the user types "delete {tab} 1" instead of using a space, then the `split(" ")` won't work. Maybe we could consider splitting by whitespace using regex.

Reply:
Sounds good, will implement